packages/components/src/toolbar-button/index.js

```diff
@@ -19,8 +19,8 @@ function ToolbarButton( {
 	className,
 	isActive,
 	isDisabled,
-	extraProps,
```

Comment:
Just noting that `extraProps` is already in use in some places in Gutenberg:
https://github.com/WordPress/gutenberg/blob/17d509ba2512326409244c8fe70739f68d0df018/packages/editor/src/components/block-settings-menu/index.js#L46-L57

So changes proposed will be braking for the components which depend on `extraProps` prop.